src/antd/SliderSingle.tsx

```diff
@@ -0,0 +1,52 @@
+import { ReactElement } from "react"
+import { ControlProps, RendererProps } from "@jsonforms/core"
+import { Slider as AntdSlider, SliderSingleProps as AntdSliderSingleProps } from "antd"
+import { coerceToInteger, coerceToNumber, decimalToPercentage } from "../controls/utils"
+
+type Slider = ReactElement<typeof AntdSlider>
+type SliderSingleProps = AntdSliderSingleProps & RendererProps & ControlProps
+
+export const SliderSingle = (props: SliderSingleProps): Slider => {
```

Comment:
why do we need to name the return type at all?

Reply:
I did this because antd has two different sets of props depending on the type of `Slider` you want. `SliderSingle` means there is a single set point on the slider instead of having two set points (see the difference [here](https://ant.design/components/slider) in the first example). Unfortunately, they are both `Slider`s though. We can just call this a `Slider` to stay true to antd and if we add support for the double slider, we can address at that time.